ansible/roles/common/tasks/main.yml

```
@@ -0,0 +1,136 @@
+- name: Create application directories
+  file:
+    path: "{{ item }}"
+    state: directory
+    owner: "{{ project_user }}"
+    group: "{{ project_user }}"
+    mode: '0755'
+  loop:
+    - "{{ project_path }}"
+    - "{{ project_path }}/logs"
+    - "{{ project_path }}/backups"
+    - "/home/{{ project_user }}/webhooks" 
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)